packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/java/integrationsteps.md

```diff
@@ -2,20 +2,40 @@
 
```

Comment:
For some reason, this file is not showing up when I build the site locally. Please double check this file is working for you locally before publishing.

Reply:
Checked. It definitely shows up for me. Weird.